packages/nimble-components/src/anchor/tests/anchor-matrix.stories.ts

```diff
@@ -85,3 +85,12 @@ export const textWrapping: StoryFn = createStory(
     </p>
     `
 );
+
+export const inlineSpacing: StoryFn = createStory(
+    html`<style>
+        * {
+            font: var(${bodyFont.cssCustomProperty});
+        }
```

Comment:
stories aren't encapsulated in shadow roots and weird things can happen with too broad styles. Please write the style in a way that won't conflict with other things. Maybe add a wrapper span around the HTML with a class and target the class instead of targeting all elements.